config/kubernetes/staging/ingress.yml

```diff
@@ -6,8 +6,6 @@ metadata:
   annotations:
     external-dns.alpha.kubernetes.io/set-identifier: ingress-staging-laa-apply-for-criminal-legal-aid-staging-green
     external-dns.alpha.kubernetes.io/aws-weight: "100"
-    nginx.ingress.kubernetes.io/auth-type: basic
-    nginx.ingress.kubernetes.io/auth-secret: secrets-staging
     nginx.ingress.kubernetes.io/server-snippet: |
       if ($host = 'laa-apply-for-criminal-legal-aid-staging.apps.live.cloud-platform.service.justice.gov.uk') {
         return 301 https://laa-apply-for-criminal-legal-aid-staging.apps.cloud-platform.service.justice.gov.uk;
```

Comment:
Do we want to remove the line `auth_basic off;` @ Line 19 as well? If we don't have it then is it confusing this is still here?

I guess if we ever put it back on then the security.txt will need to have it off so perhaps it is okay to leave. Up to you - though I would just flag as it cuaght my eye 👀

Reply:
I left that on purpose as the likes of enabling back http basic auth and forgetting to also add this annotation to the snippet are high 😄 
It does no harm at all leaving it.